config/internal/apiserver/deployment.yaml.tmpl

```diff
@@ -104,6 +104,8 @@ spec:
               value: "{{.ObjectStorageConnection.Host}}"
             - name: MINIO_SERVICE_SERVICE_PORT
               value: "{{.ObjectStorageConnection.Port}}"
+            - name: MINIO_SERVICE_SERVICE_ENDPOINT
```

Comment:
See https://github.com/opendatahub-io/data-science-pipelines-operator/pull/479/files#r1396485621, but i don't think we need this (and therefore need to un-update the functests as well)

Reply:
Done, un-updated the functests.